content/knowledge_base/glossary.md

```diff
@@ -0,0 +1,141 @@
+# Glossary
+
+## General Blockchain Concepts
+
+### Consensus
+
+TODO: add
+
+### Blockchain
+
+TODO: add
+
+### Gossip Algorithms
+
+TODO: add
+
+### Liveness Threshold
+
+The number of malicious participants that can be tolerated before the consensus protocol stops functioning.
+
+### Nakamoto Type of Consensus
+
+TODO: introduce (for BTC)
+A consensus algorithm that provides a probabilistic safety guarantee; its decisions may revet with some probability ε.
+A permissionless protocol allowing each node to join or leave the system at any time.
+TODO: finish
+
+### Safety
+
+TODO: add
+
+## Avalanche Concepts
+
+### Ancestor Set
+
+TODO: add
+
+### Chit
+
+A transaction receives a chit (boolean value) when the majority (quorum) of queried nodes respond with an approval for that transaction.
+
+### Confidence
+
+The sum of transaction/vertex's decendants cihts and its own chit.
+The chit value is a result of a one-time query for its associated transaction and becomes immutable afterwards.
+Because chit values can be 0 or 1 - c ∈ {0,1} confidence values are monotonic.
+
+### Consecutive Success Number
+
+The number of times a transaction or its decendant received an approval from majority (quorum) of queried nodes.
+TODO: add more thorough description
+
+### Progeny
+
+TODO: add
+
+### Slush
+
+TODO: add
+
+## Other
+
+### Graph
+
+TODO: introduce the idea in one sentence without technicals
+
+A graph is pair of sets G = (V, E).
```

Comment:
I feel like this section about the simple graph could simply be a Wikipedia link or something similar? I'm not sure it brings value for us to define what a graph is in our knowledge base since it is not very specific to our domain IMO, and most people already know about it.

Defining the directed graph makes a lot of sense to me though.

Reply:
The wikipedia description of the graph is not clear and full, I'll find another one

Reply:
Removed the definition and added a link that will do https://github.com/optakt/optakt.github.io/pull/13/commits/d9b5fd4dbe3d581029770f925beabfad5d4608bc#diff-55a59bbede67c988e24e88ffb8a7b56d20454b9ab95a3fd8567e790585338dfbR109
But I still dont like it ...